[BARX-1655] Add registryMigrationMode for per-site DD_REGISTRY_OVERRIDE_* env vars (#2421)
DataDog/helm-charts: charts/datadog-operator/CHANGELOG.md

# Changelog

## 2.19.0-dev.6

> * Add `registryMigrationMode` to control gradual migration of agent image pulls to registry.datadoghq.com. Defaults to "auto", which currently enables the ASIA region override (ap1.datadoghq.com). More sites will be enabled in future releases. Set to "" to disable. ([#2421](https://github.com/DataDog/helm-charts/pull/2421)).

Reviewer: ```suggestion
* Add `registryMigrationMode` to control gradual migration of Agent image pulls to `registry.datadoghq.com`. Defaults to "auto", which currently enables this behaviour for the AP1 datacenter (when `global.site: ap1.datadoghq.com`). More sites will be enabled in future releases. Set it to "" to disable. ([#2421](https://github.com/DataDog/helm-charts/pull/2421)).
```
